storeshield/tests/mock/file_storage.py

```
@@ -0,0 +1,37 @@
+        return file_path
+
+    async def download(self, file_path: str) -> BytesIO:
+        if file_path not in self._storage:
+            raise FileNotFoundError(f"File {file_path} not found")
+        return BytesIO(self._storage[file_path])
+
+    async def delete(self, file_path: str) -> None:
+        self._storage.pop(file_path, None)
+
+    # Test helper methods
+    def get_stored_content(self, file_path: str) -> Optional[bytes]:
```

Ruff (0.8.2)

Undefined name `Optional`

(F821)